articles/container-apps/TOC.yml

```diff
@@ -354,6 +354,8 @@
           href: waf-app-gateway.md
         - name: Enable User Defined Routes (UDR)
           href: user-defined-routes.md
+        - name: Use path-based routing
```

Comment:
It's not just path routing, it also supports hosts and will eventually support other types like headers.

Better to call it ingress routing or app routing.